actors/builtin/multisig/multisig_test.go

```diff
@@ -1095,21 +1123,41 @@ func (h *msActorHarness) constructAndVerify(rt *mock.Runtime, numApprovalsThresh
 	rt.Verify()
 }
 
-func (h *msActorHarness) propose(rt *mock.Runtime, to addr.Address, value abi.TokenAmount, method abi.MethodNum, params []byte) {
+func (h *msActorHarness) propose(rt *mock.Runtime, to addr.Address, value abi.TokenAmount, method abi.MethodNum, params []byte, out runtime.CBORUnmarshaler) {
 	proposeParams := &multisig.ProposeParams{
 		To:     to,
 		Value:  value,
 		Method: method,
 		Params: params,
 	}
-	rt.Call(h.a.Propose, proposeParams)
+	ret := rt.Call(h.a.Propose, proposeParams)
+	proposeReturn, ok := ret.(*multisig.ProposeReturn)
+	if !ok {
+		h.t.Fatalf("unexpected type returned from call to Propose")
+	}
+	if proposeReturn.Applied {
+		// get the last message sent and ensure it matches the exit code
+		msg := rt.LastMessageSent()
+		assert.Equal(h.t, proposeReturn.Code, msg.ExitCode())
+		assert.NoError(h.t, msg.SendReturn().Into(out))
```

Comment:
This isn't quite right. We want to deserialize the proposeReturn into the output.

This observation was triggered by the addition of LastMessageSent() and exposing the ExpectedSend. I think we should be able to avoid that altogether.

Reply:
`LastMessageSent` was added to address the (now removed) TODO:
```
// TODO In a follow-up, this method should also verify the return value from Approve contains the exit code prescribed in ExpectSend.
// exercise both un/successful sends.
```

Reply:
It could, but probably better would be to just return it from `rt.Call`, like the return value.

Reply:
Wait, no. I mean it should be parallel with the return value, so proposeReturn.Code returned from this `propose` method. The caller (who set up the ExpectSend, if any) will know the expected value and can assert it if relevant, along with checking for the expected `out`.

Reply:
How would that work? The mockrt's Call method doesn't catch aborts -- I think we'd need a new method for that.